bench/Autofac.Benchmarks/Decorators/DecoratorBenchmarkBase.cs

```diff
@@ -8,18 +8,30 @@ public abstract class DecoratorBenchmarkBase<TCommandHandler>
     {
         protected IContainer Container { get; set; }
 
+        [Benchmark(Baseline =true)]
+        public virtual void Baseline()
```

Comment:
I updated the base for all of the decorators to carry the same lifetime scope baseline and work, just so everything was an apples-to-apples comparison.